core/object.go

```diff
@@ -27,6 +27,9 @@ var OBJ_ENCODING_STACKREF uint8 = 3
 var OBJ_TYPE_BITSET uint8 = 2 << 4 // 00100000
 var OBJ_ENCODING_BF uint8 = 2      // 00000010
 
+var OBJ_TYPE_BYTEARRAY uint8 = 1 << 6
+var OBJ_ENCODING_BYTEARRAY uint8 = 4
+
```

Comment:
Please add this after OBJECT_TYPE_JSON AND OBJ_ENCODING_JSON.
Also, to maintain consistency, let's initialize as `OBJ_TYPE_BYTEARRAY = 4 << 4`